PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Please include a summary of the changes and the related issue.
+
+Fixes # (issue)
+
+## Type of change
+
+Please select all options that apply to this change.
+
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] Documentation update
+
+# Checklist:
+
+Please select all options that was done as part of this change.
+
+- [ ] My code follows the style guidelines of this project
```

Comment:
remove `guidelines`, and just state `My code follows the style of this project`

Reply:
updated